react-native/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["../react/.eslintrc.json", "../base/.eslintrc.json"],
```

Comment:
Definitely include jest in the react-native one. React-Native now ships with Jest in the box and we use Jest for all the mobile apps. I prefer not having multiple configs when I can only use one.